playground/frontend/playground_components/lib/src/cache/example_cache.dart

```diff
@@ -170,9 +170,9 @@ class ExampleCache extends ChangeNotifier {
 
       return Example.fromBase(
         example,
-        source: exampleData[0],
-        outputs: exampleData[1],
-        logs: exampleData[2],
+        files: exampleData[0] as List<SnippetFile>,
```

Comment:
0, 1, 2, 3 seem like magic numbers.

Reply:
- Will be addressed in https://github.com/apache/beam/issues/24305